dhcpv4/server4/server_test.go

```diff
@@ -73,15 +73,15 @@ func setUpClientAndServer(t *testing.T, iface net.Interface, handler Handler) (*
 		IP:   loAddr,
 		Port: randPort(),
 	}
-	s, err := NewServer(&saddr, handler)
+	s, err := NewServer(iface.Name, &saddr, handler)
```

Comment:
How about doing:

```
s, err := NewServer("", &saddr, handler)
```

to avoid binding to the interface in the test.

```
server_test.go:78: cannot bind to interface lo: operation not permitted
```